Exploration of the MNIST1000 Dataset.ipynb

```diff
@@ -0,0 +1,475 @@
+{
```

Comment:
<p>I initially left some comments about the reproducibility of this approach. Let's talk more about it later tonight! I addressed it in my PRs that got merged.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/akaashsidhu/SM_project/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>